Ordering/OrderManagement/OrderFramework.cs

```diff
@@ -0,0 +1,64 @@
+﻿using Ordering.Common.Interfaces;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+
+namespace Ordering.BusinessLogic.OrderManagement
+{
+    class OrderFramework:IOrderManager
```

Comment:
Framework is too big of a name for a class --> OrderManager?

Reply:
Sure, I can make that change.